solr/core/src/test/org/apache/solr/response/transform/TestChildDocTransformerHierarchy.java

```diff
@@ -135,6 +135,37 @@ public void testParentFilterLimitJSON() throws Exception {
     );
   }
 
+  @Test
+  public void testLiveDocs() throws Exception {
```

Comment:
```suggestion
  public void testWithDeletedChildren() throws Exception {
```

"liveDocs" is Lucene level internal terminology.